.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
+php:
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+before_install:
+  - composer self-update
+
+install:
+  - composer install --prefer-source --no-interaction --no-progress
```

Comment:
Well, if I am not mistaken, caching the Composer cache files won't work when installing from source instead of from dist.